jobs/JGLOBAL_PREP_SNOW_OBS

```diff
@@ -1,8 +1,20 @@
 #! /usr/bin/env bash
 
 source "${HOMEgfs}/ush/preamble.sh"
-export DATA=${DATA:-${DATAROOT}/${RUN}snowanl_${cyc}}
-source "${HOMEgfs}/ush/jjob_header.sh" -e "prepsnowobs" -c "base prepsnowobs"
+if (( 10#${ENSMEM:-0} > 0 )); then
+  source "${HOMEgfs}/ush/jjob_header.sh" -e "epsn" -c "base prepsnowobs epsn"
+  export CASE=${CASE_ENS}
+else
+  source "${HOMEgfs}/ush/jjob_header.sh" -e "prepsnowobs" -c "base prepsnowobs"
+fi
+
+# Restart conditions for GFS cycle come from GDAS
+rCDUMP=${RUN}
+[[ ${RUN} == "gfs" ]] && export rCDUMP="gdas"
+
+# Get OBS for ENKFGDAS run from GDAS
+oCDUMP=${RUN}
```

Comment:
so we don't export oCDUMP if the run if gdas?

Reply:
Do we need export `oCDUMP` here? Both rCDUMP and oCDUMP are used just locally.

Reply:
sorry I misread this line, but if they are only used internally then why export rCDUMP above on line 13?

Reply:
I reused this by following the old version `jobs/JGLOBAL_FORECAST` for the GFS cycle to get the restart conditions. But I just checked and found they were modified as below: 

```
# Restart conditions for GFS cycle come from GDAS
rCDUMP=${RUN}
export rCDUMP="${RUN/gfs/gdas}"
```

Do I need to make the corresponding changes?